priv/repo/seeds.exs

```diff
@@ -57,3 +57,14 @@ CoursePlanner.Repo.delete_all CoursePlanner.User
       visible: true
     })
 |> CoursePlanner.Repo.insert!
+
+%CoursePlanner.SystemVariable{}
+|> CoursePlanner.SystemVariable.changeset(
+    %{
+      key: "ATTENDANCE_DESCRIPTORS",
```

Comment:
what do you think of "ATTENDANCE_DESCRIPTIONS" instead? `DESCRIPTORS` sounds kinda technical to me